reconcile/openshift_users.py

```diff
@@ -101,6 +101,8 @@ def fetch_desired_state(
     oc_map: Optional[OCMap], enforced_user_keys: Any = None
```

Comment:
Does it make sense to remove the `Optional` here?

Reply:
It might make sense, but Im not 100% sure. This function is also used by other integrations, like `github_org` or `slack-usergroups`. Kept the interface here stable for compatibility.